src/current/_includes/releases/v23.2/v23.2.0-alpha.1.md

```diff
@@ -0,0 +1,983 @@
+
+  - `BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED`
+  - `SET TRANSACTION ISOLATION LEVEL READ COMMITTED`
+  - `SET default_transaction_isolation = 'read committed'`
+  - `SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL READ COMMITTED`
+  [#110624][#110624]
+
+- The [cluster setting](https://www.cockroachlabs.com/docs/v23.2/cluster-settings) `sql.metrics.statement_details.gateway_node.enabled` now defaults to false, to reduce the number of rows generated in SQL Statistics pages. [#107788][#107788]
+- The default value for the [`ttl_job_cron`](https://cockroachlabs.com/docs/v23.2/row-level-ttl) table storage parameter is now `@daily` rather than `@hourly`. This parameter controls the default recurrence of the row-level TTL job. As part of this change, the output of the `SHOW CREATE TABLE` statements now include the `ttl_cron_job` parameter only if it is explicitly set. [#110623][#110623]
+
+<h3 id="v23-2-0-alpha-1-operational-changes">Operational changes</h3>
+
+- Removed a timeseries metric that has not been reported for several versions. [#100524][#100524]
+- Added two new metrics, `range.snapshots.(send|recv)-queue-bytes`, to track the total size of all snapshots waiting in the snapshot queue. [#100942][#100942]
+- Exposed a new metric `storage.compactions.duration`, computed by the storage engine, that provides the cumulative time the storage engine has spent in compactions. This duration may exceed time elapsed, because of concurrent compactions, and may be useful in monitoring compaction concurrency. [#103670][#103670]
+- The default value for the `--max-sql-memory` parameter of the [`cockroach demo` command]({% link v23.2/cockroach-demo.md %}) has been increased from 128 MiB to 256 MiB. [#103642][#103642]
+- Two new store metrics, `range.snapshots.cross-region.sent-bytes` and `range.snapshots.cross-region.rcvd-bytes`, were added to track the aggregate of snapshot bytes sent from and received at a store across different regions. Note that these metrics require the nodes' localities to include a “region” tier key. If a node lacks this key but is involved in cross-region batch activities, an error message will be logged. [#104111][#104111]
+- Added new store metrics to track the aggregate of snapshot bytes sent from and received at a store across different zones.
+
+  - `range.snapshots.cross-zone.sent-bytes`
+  - `range.snapshots.cross-zone.rcvd-bytes`
+
+   For accurate metrics, follow these recommendations:
+
+  - Configure region and zone tier keys consistently across nodes.
+  - Within a node locality, ensure unique region and zone tier keys.
+  - Maintain consistent configuration of region and zone tiers across nodes.
```

Comment:
It's possible that this is how the commit should be mapped/formatted in the release notes?

But the notes for this one commit ([8f3e328a00a5d388a8cfd1664756c343e43a6d3b](https://github.com/cockroachdb/cockroach/pull/104417/commits/8f3e328a00a5d388a8cfd1664756c343e43a6d3b))

Is getting shifted left / having multiple additional bullets (instead of it being nested until the main bullet for the PR)

![image](https://github.com/cockroachdb/docs/assets/3051672/545f1256-2f8b-4c36-8bdc-05ae425f4ff4)

Reply:
update: okay, I see this happening for a few commits, so perhaps this is expected / fine.

Reply:
There were some formatting issues but I think they are fixed now. PTAL again.

Reply:
LGTM - thanks!